aspnetcore/includes/managed-identities.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. If you're connecting to Azure SQL, [Managed Identities for Azure resources](/sql/connect/ado-net/sql/azure-active-directory-authentication#using-managed-identity-authentication) is the recommended authentication method.
```

Comment:
Where is this include used?

Reply:
It is not, I may remove it.  I figured at first that when relying on just the warnings those were the two cases.  Sorry for the confusion.

Reply:
Removed this 2nd include.